### PR TITLE
operators t8c (42.10.0)

### PR DESCRIPTION
**Pull request by esara**
Signed-off-by: esara <endre.sara@turbonomic.com>

### CODE REVIEW

**Comment by mvalarh** [comment]
@esara can you prepend hostname (docker.io, ....) for turbonomic/t8c-operator:42.10?

**Comment by esara** [comment]
> @esara can you prepend hostname (docker.io, ....) for turbonomic/t8c-operator:42.10?

@mvalarh this bundle pulls from the redhat registry already - see the image reference - I dont want to change it to dockerhub for the ecosystem operators
https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/1285/files#diff-e7e97adb8ee2f0ba2f4f268a38079bbc5fca00298845dca9e81a2147214b35fdR58

are you sure that this is a problem?

**Review by mvalarh — CHANGES_REQUESTED** [review]

File: operators/t8c/42.10.0/t8c-operator.v42.10.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,199 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"charts.helm.k8s.io/v1","kind":"Xl","metadata":{"name":"xl-release"},"spec":{"global":{"tag":"8.5.3"}}}]'
+    capabilities: Basic Install
+    categories: Monitoring
+    certified: "false"
+    containerImage: turbonomic/t8c-operator:42.10
```
[the remark this answers is not in the document]
Comment (esara) [reply operators/t8c/42.10.0/t8c-operator.v42.10.0.clusterserviceversion.yaml]: of course, sorry I missed the annotation

**Pull request merged**